packages/web/src/components/common/select-tab/SelectTab.tsx

```diff
@@ -1,24 +1,45 @@
-import { SelectButton, SelectTabWrapper } from "./SelectTab.styles";
+import {
+  SelectButton,
+  ChartSelectButton,
+  SelectTabWrapper,
+} from "./SelectTab.styles";
 import { cx } from "@emotion/css";
 interface SelectTabProps {
   selectType: string;
   list: string[];
+  className?: string;
   onClick: (type: string) => void;
 }
 
-const SelectTab: React.FC<SelectTabProps> = ({ selectType, list, onClick }) => {
+const SelectTab: React.FC<SelectTabProps> = ({
+  selectType,
+  list,
+  onClick,
+  className,
+}) => {
   return (
     <SelectTabWrapper>
-      {list.map((type, idx) => (
-        <SelectButton
-          key={idx}
-          className={cx({ selected: type === selectType })}
-          onClick={() => onClick(type)}
-          disabled={type === selectType}
-        >
-          {type}
-        </SelectButton>
-      ))}
+      {list.map((type, idx) =>
+        className === "chart-tab" ? (
```

Comment:
I don't think it's explicit to handle rendering conditions by class name.
What about using explicit `type` or `enum`?

Reply:
Rendering by type or enum would be nice.